mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -96,242 +54,179 @@ def run(self) -> None:
         - Rescheduling; when a task has passed its deadline, we need to
         reschedule it.
         """
-        # Scan profile mutations
-        self.listeners["scan_profile_mutations"] = clients.ScanProfileMutation(
+        self.listeners["mutations"] = clients.ScanProfileMutation(
             dsn=str(self.ctx.config.host_raw_data),
-            queue=f"{self.organisation.id}__scan_profile_mutations",
-            func=self.push_tasks_for_scan_profile_mutations,
+            queue="scan_profile_mutations",
+            func=self.process_mutations,
             prefetch_count=self.ctx.config.rabbitmq_prefetch_count,
         )
 
-        self.run_in_thread(
-            name=f"BoefjeScheduler-{self.scheduler_id}-mutations",
-            target=self.listeners["scan_profile_mutations"].listen,
-            loop=False,
-        )
-
-        # New Boefjes
-        self.run_in_thread(
-            name=f"BoefjeScheduler-{self.scheduler_id}-new_boefjes",
-            target=self.push_tasks_for_new_boefjes,
-            interval=60.0,
-        )
-
-        # Rescheduling
-        self.run_in_thread(
-            name=f"scheduler-{self.scheduler_id}-reschedule", target=self.push_tasks_for_rescheduling, interval=60.0
-        )
+        self.run_in_thread(name="BoefjeScheduler-mutations", target=self.listeners["mutations"].listen, loop=False)
+        self.run_in_thread(name="BoefjeScheduler-new_boefjes", target=self.process_new_boefjes, interval=60.0)
+        self.run_in_thread(name="BoefjeScheduler-rescheduling", target=self.process_rescheduling, interval=60.0)
 
         self.logger.info(
-            "Boefje scheduler started for %s",
-            self.organisation.id,
-            organisation_id=self.organisation.id,
-            scheduler_id=self.scheduler_id,
-            item_type=self.queue.item_type.__name__,
+            "Boefje scheduler started", scheduler_id=self.scheduler_id, item_type=self.queue.item_type.__name__
         )
 
-    @tracer.start_as_current_span("boefje_push_tasks_for_scan_profile_mutations")
-    def push_tasks_for_scan_profile_mutations(self, body: bytes) -> None:
+    @tracer.start_as_current_span("process_mutations")
+    def process_mutations(self, body: bytes) -> None:
         """Create tasks for oois that have a scan level change.
 
         Args:
             mutation: The mutation that was received.
         """
-        # Convert body into a ScanProfileMutation
-        mutation = ScanProfileMutation.model_validate_json(body)
-
-        self.logger.debug(
-            "Received scan level mutation %s for: %s",
-            mutation.operation,
-            mutation.primary_key,
-            ooi_primary_key=mutation.primary_key,
-            organisation_id=self.organisation.id,
-            scheduler_id=self.scheduler_id,
-        )
-
-        # There should be an OOI in value
-        ooi = mutation.value
-        if ooi is None:
+        try:
```

Comment:
I'd rather keep the try except for the ValidationError smaller and only around the actual validation code.